rfd/0119/README.md

```diff
@@ -255,25 +190,23 @@ two new message types, `SVP_ROUTE_REQ` and `SVP_ROUTE_ACK`:
 
 #### SVP_ROUTE operation
 
-When portolan receives an `SVP_ROUTE_REQ`, it will query the `portolan_vnet_routes` [moray bucket] for all objects containing the combination of `srr_vnetid,srr_vlanid`.  It will then check each matching object to see if the `srr_srcip` and `srr_dstip` are included in the object's `subnet` and `r_subnet` respectively.  If a match is found portolan will fill out the `SVP_ROUTE_ACK` message and return it to the caller.
+When portolan receives an `SVP_ROUTE_REQ`, it will query the
+`portolan_vnet_routes` [moray bucket] for all objects containing the
+combination of `srr_vnetid,srr_vlanid`.  It will then check each matching
+object to see if the `srr_srcip` and `srr_dstip` are included in the object's
+`subnet` and `r_subnet` respectively.  If a match is found portolan will fill
+out the `SVP_ROUTE_ACK` message and return it to the caller.
 
 
 
 #### Shootdowns
 
-```
+The SVP_R_LOG_REQ command will include, for version 2, SVP_LOG_ROUTE
+data, in addition to version 1's SVP_LOG_VL2 and SVP_LOG_VL3.
```

Comment:
Are SVP_LOG_ROUTE structs defined anywhere?

Reply:
Not yet, but they should be.